src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -242,6 +249,11 @@ bool MulticopterLandDetector::_get_maybe_landed_state()
 		return true;
 	}
 
+	// Only trigger in RW mode
+	if (!_vehicle_status.is_rotary_wing) {
+		return false;
+	}
```

Comment:
this will cause that we will never have landed:
https://github.com/PX4/Firmware/blob/870f8ae39676bedfd3cf17957ffd6dfefd63d65e/src/modules/land_detector/MulticopterLandDetector.cpp#L321